src/cli/commands/load.js

```diff
@@ -31,14 +31,9 @@ export default class PluginGraphCommand extends Command {
 
   static args = [
     {
-      name: "repo_owner",
+      name: "repo",
       required: true,
-      description: "owner of the GitHub repository for which to fetch data",
-    },
-    {
-      name: "repo_name",
-      required: true,
-      description: "name of the GitHub repository for which to fetch data",
+      description: "the GitHub repo to load, represented as $OWNER/$NAME. ",
```

Comment:
Let’s write `OWNER/NAME` instead of `$OWNER/$NAME`; the former is
standard man-page syntax.

Let’s also remove the trailing `". "` from this string, for consistency.

Reply:
Done.